highbury/chain.json

```diff
@@ -0,0 +1,99 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "highbury",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://www.fury.black/",
+  "pretty_name": "Highbury",
+  "chain_id": "highbury_710-1",
+  "bech32_prefix": "fury",
+  "daemon_name": "fury",
+  "node_home": "$HOME/.fury",
+  "slip44": 459,
```

Comment:
this is Kava's slip44, while FURY was registered as 710

Reply:
Yes, it will be kava's slip44.